app/main/db/queries.py

```diff
@@ -560,3 +560,46 @@ def get_all_transferring_bodies():
     for body in Body.query.all():
         bodies.append(body.Name)
     return bodies
+
+
+def get_breadcrumb_values(
+    transferring_body_id=None, series_id=None, consignment_id=None
+):
+    dict_values = {}
+    if transferring_body_id:
+        transferring_body = (
+            Body.query.filter(Body.BodyId == transferring_body_id).first()
+        ).Name
+        dict_values[0] = {"transferring_body": transferring_body}
+    elif series_id:
+        result = (
+            db.session.query(Body.BodyId, Body.Name, Series.Name)
```

Comment:
[Suggestion] Instead of session.query you could use just do

```
series = Series.query.get(series_id)
body = series.body
```

I think it is more concise/readable, what do you think?

Reply:
updated logic to follow as mentioned above